karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/oasExamples.feature

```diff
@@ -0,0 +1,410 @@
+    * header Content-Type = "application/json-patch+json"
+    * header If-Match = karate.response.header('etag')
+    * def extensionVal = 
+    """
+        {
+          "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-ContactPreference",
+          "extension": [
+          {
+            "url": "PreferredContactMethod",
+            "valueCodeableConcept": {
+                "coding": [{
+                        "code": "7",
+                        "display": "Sign language",
+                        "system": "https://fhir.nhs.uk/R4/ValueSet/UKCore-PreferredContactMethod"
+                    }
+                ]
+            }
+          },
+          {
+            "url": "PreferredContactTimes",
+            "valueString": "13:00"
+          }
+        ]
+      }
+    """
+    * path 'Patient', nhsNumber
+    * request 
+     """ 
+        {
+            "patches": [
+                {
+            "op": "replace",
+            "path": "/extension/5",
+            "value": "#(extensionVal)"
+            }
+            ]
+        }
+    """
+    * method patch
+    * status 200
+    * match response.extension[5] contains extensionVal
+    * path 'Patient', nhsNumber
+    * method get
+    * status 200
+    * def originalVersion = parseInt(response.meta.versionId)
+    * configure headers = call read('classpath:auth/auth-headers.js')     
+    * header Content-Type = "application/json-patch+json"
+    * header If-Match = karate.response.header('etag')  
+    * path 'Patient', nhsNumber
+    * request 
+     """ 
+        {
+            "patches": [
+                {
+            "op": "remove",
+            "path": "/extension/5",
+            }
+            ]
+        }
+    """
+    * method patch
+    * status 200
+    * match response.extension !contains extensionVal 
+    
+Scenario:  Healthcare worker can remove Single Contact Preference
+    * def nhsNumber = '9000000009'
+    * path 'Patient', nhsNumber
+    * method get
+    * status 200
+    * def originalVersion = parseInt(response.meta.versionId)
+    * configure headers = call read('classpath:auth/auth-headers.js')     
+    * header Content-Type = "application/json-patch+json"
+    * header If-Match = karate.response.header('etag')
+    * def extensionURL = "PreferredWrittenCommunicationFormat"
+    * path 'Patient', nhsNumber
+    * request 
+     """ 
+        {
+            "patches": [
+              { 
+                "op": "test",
+                "path": "/extension/5/extension/0/url",
+                "value": "#(extensionURL)"
+              },
+              { 
+                "op": "remove",
+                "path": "/extension/5/extension/2"
+              }
+            ]
+        }
+    """
+    * method patch
+    * status 200
+    * def extensions = response.extension[5].extension.length
+    * if (extensions > 2) { karate.fail('Single Contact Preference not been removed from /extension/5/extension/2.')}
```

Comment:
Might be worth adding a few more assertions here, such as extension[5] is still the one we care about, and possibly those other two extensions are the one's we expect.